ci-operator/step-registry/assisted/baremetal/operator/publish/assisted-baremetal-operator-publish-commands.sh

```diff
@@ -137,25 +137,6 @@ else
 fi
 echo "   operator version: ${OPERATOR_VERSION} will replace version ${PREV_OPERATOR_VERSION}"
 
-echo
-echo "## Determine operator versions we skip"
-if [[ "${BUNDLE_CHANNELS}" == *"alpha"* ]]; then
-    echo "   since we are updating alpha channel we will skip every operator version"
-    skips=$(find "${CO_OPERATOR_DIR}"/* -maxdepth 1 -type d)
-    OPERATOR_SKIPS=""
-    for version in ${skips}; do
-        OPERATOR_SKIPS="${OPERATOR_SKIPS} assisted-service-operator.v${version##*\/}"
-    done
-else
-    echo "   use previous version to determine the versions we skip"
-    OPERATOR_SKIPS=$(yq eval ".spec.skips | .[]" "${CO_OPERATOR_DIR}/${PREV_OPERATOR_VERSION}/${CSV}")
-    OPERATOR_SKIPS="${OPERATOR_SKIPS} assisted-service-operator.v${PREV_OPERATOR_VERSION}"
-fi
-echo "   skipping these operator versions: "
-for version in ${OPERATOR_SKIPS}; do
-    echo "    - ${version}"
-done
-
```

Comment:
let's also remove the part below `echo "   adding spec.skips" ...`

Reply:
Thank you, missed that